chart/values.yaml

```diff
@@ -15,6 +15,8 @@ metricsOperator:
 ##
 
 global:
```

Comment:
you need to remove the possibility to enable certManager form this file on lines 2-4, additionally, you need to adapt Chart.yaml for reflect the updated value, which is enabling or disabling the certManager dependency